platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -104,9 +103,18 @@ public void doExecute(JobExecutionContext context) throws JobExecutionException
         ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
         Session session = null;
 
-        try (PrintStream output = getPrintStream(byteArrayOutputStream)) {
+        // TODO DDF-3280 remove work-around for NPE when creating session with a null "in" parameter from a SessionFactory
+        try (PrintStream output = getPrintStream(byteArrayOutputStream);
```

Comment:
❓  Is there any reason not to move the ByteArrayOutputStream into this try-with-resources? It looks like you might be able to move the session inside as well and completely remove the `finally`.

Reply:
:+1: The finally now only contains:
```java
        } finally {
            if (session != null) {
                session.close();
            }
        }
```